app/pages/api/ask.ts

```diff
@@ -144,21 +145,35 @@ export default async function handler(
     });
 
     const gptResponse = await ChatGPT.sendMessage(
-      codeBlock`
+      stripIndent`
+      Different pieces of context are separated by "---".
       CONTEXT:
       ${context}
+
       QUESTION:
+      """
       ${question}
+      """
+
+      Include a list of "SOURCES" at the bottom of the response.
+      Only use links in the SOURCE_URLs from the conversation.
+      Only include the SOURCE_URL if the CONTENT next to it is used in the answer.
+      Format the links as Markdown links, such as [https://example.com](https://example.com).
+      ONLY include links in context information provided in this conversation section.
+      NEVER use a link not present in the CONTEXT information.
+      NEVER use a link that is not provided in the chat. NEVER make up a link.
```

Comment:
for reasons beyond my understanding, including this stuff last helps emphasize it to the LLM.

Reply:
🤣🤣🤣